addOns/beanshell/src/main/java/org/zaproxy/zap/extension/beanshell/BeanShellConsoleFrame.java

```diff
@@ -78,14 +76,11 @@ public BeanShellConsoleFrame() throws HeadlessException {
     /**
      * @param parent
      * @param modal
-     * @param extension
      * @throws HeadlessException
      */
-    public BeanShellConsoleFrame(Frame parent, boolean modal, Extension extension)
-            throws HeadlessException {
+    public BeanShellConsoleFrame(Frame parent, boolean modal) throws HeadlessException {
```

Comment:
This constructor is really the same as the other one without parameters (`parent` and `modal` aren't used), maybe it can be removed?

Reply:
Its part of the public API, though its unlikely (I don't know of any) other add-ons could use it.

I'm not against dumping it, but I guess we should have some consensus).

Reply:
It's already breaking the API with the removal of the `extension` parameter. I'm +1 to remove the whole constructor (and remove/merge initialize method as well).

Reply:
Good point